moto/sqs/responses.py

```diff
@@ -396,8 +398,30 @@ def receive_message(self):
         messages = self.sqs_backend.receive_messages(
             queue_name, message_count, wait_time, visibility_timeout, message_attributes
         )
+
+        attributes = {
+            "approximate_first_receive_timestamp": False,
+            "approximate_receive_count": False,
+            "sender_id": False,
+            "sent_timestamp": False,
+        }
+
+        if "All" in attribute_names:
```

Comment:
These if-statements can probably become a for-loop, using the `moto.utils.underscore_to_camelcase` method to streamline the conversion